test/services/dumper/dumper-sql.test.js

```diff
@@ -12,58 +14,54 @@ function cleanOutput() {
 
 describe('services > dumper > SQL', () => {
   describe('database MySQL', () => {
-    describe('handling /models/index.js file', () => {
-      async function dump() {
-        const config = {
-          appName: 'test-output/mysql',
-          dbDialect: 'mysql',
-          dbConnectionUrl: 'mysql://localhost:8999',
-          ssl: false,
-          dbSchema: 'public',
-          appHostname: 'localhost',
-          appPort: 1654,
-        };
-
-        const dumper = await new Dumper(config);
-        await dumper.dump({});
-      }
-
-      it('should force type casting for boolean', async () => {
-        expect.assertions(1);
-        await dump();
-        const indexGeneratedFile = fs.readFileSync('./test-output/mysql/models/index.js', 'utf-8');
-
-        expect(indexGeneratedFile).toStrictEqual(expect.stringMatching('databaseOptions.dialectOptions.typeCast'));
-        cleanOutput();
-      });
+    async function dump() {
```

Comment:
Indentation and diff... ;( 
we may exclude tab/space, isn't it?

Reply:
I don't understand @slimee .
Look at the diff with this link: https://github.com/ForestAdmin/lumber/pull/376/files?utf8=%E2%9C%93&diff=split&w=1
There is no indentation issue here IMO.